lrsnashlib.h

```diff
@@ -65,9 +65,10 @@ void printGame(game * g);
 void setFwidth(game *g, int len);
 void initFwidth(game *g);
 void updateFwidth(game *g, int col, int pos, char *str);
+#define resetNashSolver suf(resetNashSolver)
```

Comment:
Is this meant to have `resetNashSolver` be called as `resetNashSolver_gmp`?
(Why is this needed?)

Reply:
This last commit is useless indeed, it should be removed